src/openclsim/plot/vessel_planning.py

```diff
@@ -62,28 +62,29 @@ def get_gantt_chart(
         and dicts need to be merged first into 1 overall list or dict, e.g.
         concepts = [from_site, to_site, *vessels.values()]
     activities
-        a list or dict of additional activities to be plotted, if not yet in concepts
+        a list or dict of additional activities to be plotted,
+        if not yet in concepts
     id_map
-        by default only the legend labels of activities in concepts are resolved.
-        Activities associated with vessels and sites are not resolved. id_map
-        resolves the legend labels using extra metadata:
+        by default only the legend labels of activities in concepts are
+        resolved. Activities associated with vessels and sites are not
+        resolved. id_map resolves the legend labels using extra metadata:
         * a list or dict of vessels
         * a manual id_map to resolve uuids to labels, e.g. {'uuid1':'name1'}
 
     """
     default_blockwidth = 10
 
     # unpack dict to list
-    if type(concepts) == dict:
+    if type(concepts) is dict:
```

Comment:
Nog net iets beter is instance(concepts, dict), dan werkt het ook voor alle subclasses van dict.

Reply:
thnx, wil do next time